apps/.mvn/extensions.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Enables build caching for all projects within BFD to improve build efficiency-->
+<extensions>
+  <extension>
+    <groupId>org.apache.maven.extensions</groupId>
+    <artifactId>maven-build-cache-extension</artifactId>
+    <version>1.0.0</version>
+  </extension>
+</extensions>
```

Comment:
New file, new rules! Can you adjust your local editor's configuration to add newlines at EOF?

Reply:
Added to settings